app/src/lib/helpers/pull-request-matching.ts

```diff
@@ -0,0 +1,41 @@
+import { IRemote } from '../../models/remote'
+import { repositoryMatchesRemote } from '../repository-matching'
+import { Branch } from '../../models/branch'
+import { PullRequest } from '../../models/pull-request'
+
+/**
+ * Find the pull request for this branch.
+ *
+ * The upstream remote is only used
```

Comment:
I'm not sure I understand this comment? 🤔 Or I'm having a hard time figuring out what it means in relation to the arguments.

Reply:
fixed this in d1d4c19b5